src/Marten/IDocumentSession.cs

```diff
@@ -201,12 +201,19 @@ public interface IDocumentSession : IQuerySession
         void QueueOperation(IStorageOperation storageOperation);
 
 
-        /// <summary>
-        /// Completely remove the document from this session's unit of work tracking and identity map caching
-        /// </summary>
-        /// <typeparam name="T"></typeparam>
+        /// <summary>
+        /// Completely remove the document from this session's unit of work tracking and identity map caching
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
         /// <param name="document"></param>
         void Eject<T>(T document);
+
+        /// <summary>
+        /// Completely remove all the documents of given type from this session's unit of work tracking and identity map caching
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="type"></param>
+        void EjectAllOfType(Type type);
```

Comment:
That bumps this to 2.6.0 instead of 2.5.1. No big deal. Makes tracking a little easier